README.md

```diff
@@ -1,4 +1,4 @@
-<img src="https://moveit.ros.org/assets/logo/moveit_logo-black.png" alt="MoveIt Logo" width="200"/>
+<img src="https://camo.githubusercontent.com/24beba601ee2b6d2251cc0ef1a5abe5619af4097ec3495cdee1f5a53fdbdf8dd/687474703a2f2f6d6f766569742e726f732e6f72672f6173736574732f6c6f676f2f6d6f766569745f6c6f676f2d626c61636b2e706e67" alt="MoveIt Logo" width="200"/>
```

Comment:
This is definitely not a sustainable link. Why didn't you used the one I proposed?
```suggestion
<img src="http://moveit.ros.org/assets/logo/moveit_logo-black.png" alt="MoveIt Logo" width="200"/>
```

Reply:
sorry for the misunderstanding , I have corrected it